src/common/interceptors/error-logger.interceptor.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  CallHandler,
+  ExecutionContext,
+  Injectable,
+  NestInterceptor,
+} from '@nestjs/common';
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { inspect } from 'util';
+
+@Injectable()
+export class ErrorLoggerInterceptor implements NestInterceptor {
+  controller: string;
+  method: string;
+
+  constructor(controller: string, method: string) {
+    this.controller = controller;
+    this.method = method;
+  }
+
```

Comment:
You can get controller and method from context in Interceptor. You've write a lot of code in controllers but this is much easier. For example: you can get controller - context.getClass().name;

Reply:
Yes, thx.
Fixed it in the next branch(Task-7-1)
https://github.com/anticher/nodejs-st-22-rest-api/pull/8/commits/d175abb947a060255f498ae10ccebd0dc9c49887